project-koku/masu
# Handle multiple .csv files in OCP payload

**Overview**
The OCP payload which is downloaded from the upload service will now contain a list of files containing both the normal OCP report csv and also second csv file with storage data.

This change makes the connections from Kafka Handler -> OCP Downloader -> OCP Report Processor.

For now the OCP report processor is skipping storage .csv files.  There is some temporary code that will be altered in a follow on PR when we begin processing the storage reports.

**Testing**
1.  Add an OCP provider with cluster id: my-ocp-cluster-1
2.  Generate a nise report csv and package with another storage csv using ocp_csv_package.py script
3. Upload tarball via Insights upload service
4.  Trigger masu download and verify OCP non-storage report data is processed and storage report is ignored

**Test Results**
[masu_ocp_multiple_files_ut.txt](https://github.com/project-koku/masu/files/2838230/masu_ocp_multiple_files_ut.txt)

WIP for #347
